benchmarks/entropy_adiabat/entropy_adiabat.prm

```diff
@@ -0,0 +1,186 @@
+# impossible if not for the new feature in this benchmark, which is to solve
+# the advection-diffusion equation in terms of entropy instead of temperature.
+# In pressure-entropy space there is no jump in entropy across the phase
+# transition, and the equation can be accurately solved (in fact it is trivial
+# for this example as the steady-state solution is at constant entropy
+# everywhere in the model once the warmer inflowing material has replaced the
+# initial material in the model). The resulting temperature for any given
+# pressure-entropy is looked up in the used material data table.
+
+# As a benchmark this model can only test that ASPECT's results are consistent
+# with the results of the thermodynamic modeling package that produced the
+# entropy-pressure tables for material properties (in this case HeFESTo). This
+# test is done by replacing the 'compute entropy profile' adiabatic conditions
+# plugin with the 'ascii data' plugin, which will load a precomputed adiabat
+# from HeFESTo instead of computing an adiabat inside ASPECT. The disadvantage
+# of this approach is of course that it is only possible for those adiabats
+# that have been precomputed (in this case one for 1600 K), while the 'compute
+# entropy profile' plugin can compute adiabats at arbitrary entropy.
+
+set Additional shared libraries            = ./plugins/libentropy_plugins.so
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+
+set End time                               = 2.5e8
+set Output directory                       = output-entropy-adiabat
+set Nonlinear solver scheme                = iterated Advection and Stokes
+
+
+subsection Formulation
+  set Mass conservation = projected density field
+end
+
+
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X extent = 13000
+    set Y extent = 1300000
+    set Y repetitions = 200
+  end
+end
+
+
+subsection Initial temperature model
+  # The actual initial temperature is irrelevant, because it
+  # is recalculated from the entropy during the first timestep.
+  set Model name = function
+
+  subsection Function
+    set Function expression = 1595
+  end
+end
+
+
+set Surface pressure                       = 0
+set Adiabatic surface temperature          = 1595
+
+subsection Adiabatic conditions model
+  # The 'ascii data' option does not need the additional
+  # plugin, but is limited to the available adiabat data files.
+  # 'compute entropy profile' computes arbitrary adiabats 
+  # internally, based on the data table.
+  set Model name = compute entropy profile #or: ascii data
+
+  subsection Compute entropy profile
+    # Entropy equivalent to T=1595 K according to the used table
+    set Surface entropy = 2534
+  end
+
+  subsection Ascii data model
+    set Data directory  = $ASPECT_SOURCE_DIR/data/material-model/
+    set Data file name  = entropy-table/opxtable2/opxtable2_T.ad1600.txt
```

Comment:
I am confused, this is opxtable2_T.ad1600.txt, but the file you add is data/material-model/entropy-table/opxtable/opxtable_T.ad1600.txt.